### DISCUSSION TITLE
Connection problem between firefly 3 and firefly importer

### DISCUSSION THREAD

**Discussion by omcoist** [firefly-iii/firefly-iii#7550] [Firefly III]
Somehow I can't connect between firefly and firefly import. Any idea what the problem might be? Can you please watch the screen record below?

Today is the second day of my struggle with this problem.

Video
https://github.com/firefly-iii/firefly-iii/assets/107047695/8fe85525-d150-4d3e-9295-3f2474cf3da5

İmage
![image](https://github.com/firefly-iii/firefly-iii/assets/107047695/e3dbcb62-364a-49c9-9d8f-f9596f262f4b)

**Comment by JC5** [comment]
Press [Reauthenticate] and [Start over] a few times, then try again.
